### DISCUSSION TITLE
Ordered Logistic Model

### DISCUSSION THREAD

**Discussion by gyansinha** [JuliaStats/MixedModels.jl#729] [Q&A] [answered]
Is there a way to fit ordered logistic models in MixedModels?

**Comment by palday** [comment, marked as answer]
No, ordered logit and multinomial models are not currently supported. They're on my todo list, but my todo list is very long and I'm not familiar with all the mathematical/computational details, so I would still need some time to write the code once they get to the top of todo list.

**Reply by gyansinha** [reply]
Thanks - when you do get to it, and need any help, I'd volunteer my time though I'm only somewhat competent in Julia.
